### PR TITLE
Revert "Auto stash before merge of "master" and "origin/master""

### PR DESCRIPTION
**Pull request by janghyukjin**
Reverts JungChaeMoon/git_test#1

### CODE REVIEW

**Comment by JungChaeMoon** [comment]
hmm.. i don't know but...

File: README.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 
 미치광이들
-혁진이~
-미친스트르녀
```
Comment (donghoney) [inline README.md:6]: dfsdfsdf

File: README.md [hunk 2]
```diff
@@ -2,7 +2,6 @@
...
 
+미친스트르녀
```
Comment (JungChaeMoon) [inline README.md:6]: hmm.,, 께름칙스..

**Review by shawn159 — APPROVED** [review]

**Review by donghoney — APPROVED** [review]

**Review by JungChaeMoon — APPROVED** [review]
ok.

**Pull request merged**